bgpd/bgpd.h

```diff
@@ -424,6 +424,7 @@ struct bgp {
 #define BGP_FLAG_DELETE_IN_PROGRESS       (1 << 22)
 #define BGP_FLAG_SELECT_DEFER_DISABLE     (1 << 23)
 #define BGP_FLAG_GR_DISABLE_EOR           (1 << 24)
+#define BGP_FLAG_IPV6_LINK_LOCAL_ONLY (1 << 25)
```

Comment:
please follow spacing around you

Reply:
@donaldsharp  The format of line no 427 is as per git clang-formatting. I tried changing the format just like no 426, however  clang-formatting is still throwing error.